wallet/AndroidManifest.xml

```diff
@@ -430,6 +430,13 @@
                 android:resource="@xml/file_provider" />
         </provider>
 
+        <provider
+            android:name="androidx.work.impl.WorkManagerInitializer"
+            android:authorities="${applicationId}.workmanager-init"
+            android:enabled="false"
+            android:exported="false"
+            tools:replace="android:authorities" />
```

Comment:
Turning off default worker initialization is required for hilt injection to work. Recheck all the CoroutineWorkers, everything is working fine.